arguebuf/schema/sadface.py

```diff
@@ -0,0 +1,38 @@
+import typing as t
+import arguebuf as ag
+
+NodeType = t.Literal["scheme", "atom"]
+DATE_FORMAT = "YYYY-MM-DD HH:mm:ss"
```

Comment:
The format seems to be different: [`2019-04-22T23:52:30`](https://github.com/Open-Argumentation/SADFace/blob/c17fd89e3f6355cbe9bc4bd320bb1461894e2752/examples/hangback/data.json#L35)